playbooks/tests/tasks/ceph.yml

```diff
@@ -13,3 +13,62 @@
   tasks:
   - name: cinder uses rbd_hybrid or rbd_ssd as backend
     shell: grep -P "^enabled_backends.+rbd_(hybrid|ssd)" /etc/cinder/cinder.conf
+
+- name: restart ceph osd service,and check ceph status
+  hosts: ceph_osds[0]
+  run_once: true
+  tasks:
+  - name: restart ceph osd service(rhel) 
+    shell: systemctl restart ceph-osd.target
+    when: ursula_os == 'rhel'
+
+  - name: restart ceph osd service(ubuntu)
+    shell: restart ceph-osd-all
+    when: ursula_os == 'ubuntu'
+
+  - name: check ceph cluster health
+    shell: ceph health 
+    register: result
+    until: result.stdout == 'HEALTH_OK'
+    retries: 3
+    delay: 30
+
+- name: restart ceph mon service , and check ceph status
+  hosts: ceph_monitors[0]
+  tasks:
+  - name: restart ceph mon service(rhel)
+    shell: systemctl restart ceph-mon.target
+    when: ursula_os == 'rhel'
+
+  - name: restart ceph mon service(ubuntu)
+    shell: restart ceph-mon-all
+    when: ursula_os == 'ubuntu'
+
+  - name: check ceph cluster health
+    shell: ceph health 
+    register: result
+    until: result.stdout == 'HEALTH_OK'
+    retries: 3
+    delay: 30
+
+- name: restart ceph_osds node, and check ceph status
+  hosts: ceph_osds[0]:ceph_monitors[0]
+  serial: 1
+  tasks:
+    - name: reboot ceph_osds|ceph_monitor nodes
+      shell: shutdown -r +1
```

Comment:
I am not sure, but I think you can try `command` module to see if we will fail too.